tendrl/node_agent/flows/create_cluster/ceph_help.py

```diff
@@ -0,0 +1,109 @@
+# flake8: noqa
+import socket
+import time
+
+
+def create_ceph(parameters):
+    # install the packages
+    status, mons, osds = install_packages(parameters)
+    if not status:
+        return status
+    # Configure Mons
+    cluster_mons, failed = create_mons(parameters, mons)
+    if not cluster_mons:
+        return False
+    # Configure osds
+    failed = create_osds(parameters, cluster_mons, osds)
+
+    return True
+
+
+
+def install_packages(parameters):
+    plugin = tendrl_ns.provisioner.get_plugin()
+    mons = []
+    osds = []
+    for key, value in parameters["node_configuration"].iteritems():
+        if "mon" in value["role"].lower():
+            # TODO might require fqdn instead of ip?
+            mons.append(value["provisioning_ip"])
+        else if "osd" in value["role"].lower():
+            # TODO might require fqdn instead of ip?
+            osds.append(value["provisioning_ip"])
+     task_id = plugin.install_mon(mons)
+     status = sync_task_status(task_id)
+     if not status:
+         return status, mons, osds
+     task_id = plugin.install_osd(mons)
+      status = sync_task_status(task_id)
+      if not status:
+          return status, mons, osds
+    return mons, osds
+
+def create_mons(parameters, mons):
+    created_mons = []
+    failed = []
+    plugin = tendrl_ns.provisioner.get_plugin()
+    for key, value in parameters["node_configuration"].iteritems():
+        if "mon" in value["role"].lower() and value["provisioning_ip"] in mons:
+            task_id = plugin.configure_mon(
```

Comment:
We will need changes as per new signature which takes interfaces, cluster_config etc.